lib/tasks/reference_data.rake

```diff
@@ -39,6 +39,13 @@ namespace :reference_data do
     ActiveRecord::FixtureSet.create_fixtures(Rails.root.join('db/fixtures'), 'suppliers')
   end
 
+  desc 'create prison transfer reasons'
+  task create_prison_transfer_reasons: :environment do
+    require 'active_record/fixtures'
+
+    ActiveRecord::FixtureSet.create_fixtures(Rails.root.join('db/fixtures'), 'prison_transfer_reasons')
```

Comment:
What happens if we run this with already existent transfer reasons?
Is the id going to change?
what happens if we add a new reason?

Reply:
in `AssessmentQuestions::Importer` we used a different approach, have a look.